src/plugins/fandom-wirewax.plugin.ts

```diff
@@ -237,26 +259,33 @@ class FandomWirewaxPlugin {
 			} else if (typeof event.seekTo === 'number') {
 				this.player.seek(event.seekTo);
 			}
-			// TODO: reaplace with tracking
-			console.log('wirewax seek', { event });
 		} catch (err) {
 			console.log(err);
 		}
 	};
 
 	WirewaxHotspotClickHandler: (event: HotspotClickEmbedderEventData) => void = (event) => {
-		// TODO: reaplace with tracking
-		console.log('wirewax hotspot click', { event });
+		this.tracker({
+			category: WIREWAX_CATEGORY,
+			action: HOTSPOT_CLICK_ACTION,
```

Comment:
Is this event for the clickable overlays on the Video?

Reply:
yes